solutions/devsprint-michael-douglas-2/DeliveryApp/Screens/Components/LoadingView.swift

```diff
@@ -0,0 +1,93 @@
+//
+//  LoadingView.swift
+//  DeliveryApp
+//
+//  Created by Hyago Henrique on 21/02/22.
+//
+
+import UIKit
+
+final class LoadingView: UIView {
+
+    // MARK: - View Components
+
+    private let loadingContainer: UIView = {
+        let container = UIView(frame: .zero)
+        container.translatesAutoresizingMaskIntoConstraints = false
+        container.isOpaque = true
+        container.isHidden = false
+        container.backgroundColor = .white
+        return container
+    }()
+
+    var loadingSpinner: UIActivityIndicatorView = {
+        let indicator = UIActivityIndicatorView(frame: .zero)
+        indicator.translatesAutoresizingMaskIntoConstraints = false
+        indicator.style = .large
+        indicator.isHidden = false
+        indicator.hidesWhenStopped = true
+        indicator.color = .black
+        return indicator
+    }()
+
+    var loadingMessage: UILabel = {
+        let label = UILabel(frame: .zero)
+        label.translatesAutoresizingMaskIntoConstraints = false
+        label.text = "Carregando..."
+        label.textAlignment = .center
+        label.font = UIFont.systemFont(ofSize: 18, weight: .medium)
+        label.numberOfLines = 0
+        label.textColor = .black
+        return label
+    }()
+
+    // MARK: - Super Methods
+
+    init() {
+        super.init(frame: .zero)
+
+        self.setupViews()
+    }
+
+    required init?(coder: NSCoder) {
+        fatalError("init(coder:) has not been implemented")
+    }
+
+    // MARK: - Public Methods
```

Comment:
Quando você não especifica nenhum nível de controle de acesso, o padrão é sempre `Internal`.

Artigo sobre Controle de Acesso: https://docs.swift.org/swift-book/LanguageGuide/AccessControl.html

```suggestion
    // MARK: - Internal Methods
```